block/block.go

```diff
@@ -30,3 +49,10 @@ func NewBlockRepository(db *sql.DB, dbtype types.DatabaseType) BlockRepository {
 		return nil
 	}
 }
+
+// NodeToBlock converts the coordinates from the given Node into the equivalent
+// Block position. Each block contains 16x16x16 nodes.
+func NodeToBlock(x, y, z int) (int, int, int) {
```

Comment:
Rename to `AsBlockPos` maybe?

Reply:
`AsBlockPos(x, y, z int) (int, int, int)` makes more sense, yes